### PR TITLE
Add missing stream services

### PR DESCRIPTION
**Pull request by fernandog**
@ratoaq2 @duramato

### CODE REVIEW

File: guessit/test/streaming_services.yaml [hunk 1]
```diff
@@ -0,0 +1,1148 @@
...
+  type: episode
+  video_codec: H.264
+
+? Jay.Lenos.Garage.2015.S03E02.1080p.CNBC.WEB-DL.x264-TOPKEK
+: episode: 2
+  release_group: TOPKEK
+  screen_size: 1080p
+  season: 3
+  source: Web
+  streaming_service: CNBC
```
Comment (fernandog) [inline guessit/test/streaming_services.yaml]: @duramato what is CNBC?

File: guessit/test/streaming_services.yaml [hunk 2]
```diff
@@ -0,0 +1,1148 @@
...
+? The.Amazing.Race.Canada.S03.720p.CTV.WEBRip.AAC2.0.H.264-BTW
+: audio_channels: '2.0'
+  audio_codec: AAC
+  country: CA
+  other: Rip
+  release_group: BTW
+  screen_size: 720p
+  season: 3
+  source: Web
+  streaming_service: CTV
```
Comment (fernandog) [inline guessit/test/streaming_services.yaml]: @duramato Canada Television?

File: guessit/test/streaming_services.yaml [hunk 3]
```diff
@@ -0,0 +1,1148 @@
...
+  type: episode
+  video_codec: H.264
+
+? Abidin.Dino.DDY.WEBRip.AAC2.0.H.264-BTN
+: audio_channels: '2.0'
+  audio_codec: AAC
+  other: Rip
+  release_group: BTN
+  source: Web
+  streaming_service: Digiturk Diledigin Yerde
```
Comment (fernandog) [inline guessit/test/streaming_services.yaml]: @ratoaq2 @duramato i added as `'Digiturk Dilediğin Yerde'` but the special `g` is not showing up

File: guessit/test/streaming_services.yaml [hunk 4]
```diff
@@ -0,0 +1,1147 @@
...
```
Comment (fernandog) [inline guessit/test/streaming_services.yaml]: @ratoaq2 @duramato i added as `'Digiturk Dilediğin Yerde'` but the special `g` is not showing up

**Comment by fernandog** [comment]
no idea why travis is failing. help?

**Comment by fernandog** [comment]
@duramato still missing release names for these:

.ATK.
.AUBC.
.CHGD.
.CrunchyRoll.
.DHF.
.DOCC.
.HBO.
.HBOGo.
.None.
.PLUZ.
.SHMI.
.STAN.
.SVT.
.SWER.
.SeeSo.
.UNIV.
.VMEO.
.VRV.
.WME.
.WNET.
.XBOX.
.YHOO.
.ZDF.

**Comment by sharkykh** [comment]
> no idea why travis is failing. help?

@fernandog 
https://docs.pylint.org/en/1.6.0/features.html#id7 - `too-many-statements` - default is 50 statements per test method.

You can either add `# pylint: disable=too-many-statements` above that method definition line,
or rewrite it into smaller batches (alphabetically?)

**Comment by fernandog** [comment]
@ratoaq2 test passed but its not detecting the streaming service:

any idea why it passed and why it didn' detected it ?
```
guessit -y "Other.Space.S01.1080p.YHOO.WEBRip.AAC2.0.x264-BTW"
? Other.Space.S01.1080p.YHOO.WEBRip.AAC2.0.x264-BTW
: audio_channels: '2.0'
  audio_codec: AAC
  other: Rip
  release_group: BTW
  screen_size: 1080p
  season: 1
  source: Web
  title: Other Space
  type: episode
  video_codec: H.264
```

**Comment by ratoaq2** [comment]
Looking at this list, I think it should go to the config file, like expected titles, available languages and available countries. No change required from your side, but I might base my work in this PR depending on how we're proceeding with streaming services.

**Comment by duramato** [comment]
```
Americas.Test.Kitchen.S17.720p.ATK.WEB-DL.AAC2.0.x264-BTN
Bushwhacked.Bugs.S01.AUBC.WEBRip.AAC2.0.H.264-DAWN
VICE.S05E12.1080p.HBO.WEB-DL.AAC2.0.H.264-monkee
Dix.Pour.Cent.S02.PLUZ.WEBRip.AAC2.0.H.264-TURTLE
Ulveson.och.Herngren.S01.720p.SVT.WEBRip.AAC2.0.H.264-BTN
555.S01.1080p.VMEO.WEBRip.AAC2.0.x264-BTN
Bravest.Warriors.S03.1080p.VRV.WEBRip.AAC2.0.x264-BTN
The.Late.Night.Big.Breakfast.S02.WME.WEBRip.AAC2.0.x264-BTN
Hockey.Wives.S02.WNET.WEBRip.AAC2.0.H.264-BTW
GTTV.E3.All.Access.Live.Day.1.Xbox.Showcase.Preshow.HDTV.x264-SYS
Sin.City.Saints.S01.1080p.YHOO.WEBRip.AAC2.0.x264-NTb
```

**Comment by fernandog** [comment]
Travis error:

```
pypy is not installed; attempting download
Downloading archive: https://s3.amazonaws.com/travis-python-archives/binaries/ubuntu/14.04/x86_64/-.tar.bz2
$ sudo tar xjf -.tar.bz2 --directory /
bzip2: (stdin) is not a bzip2 file.
tar: Child returned status 2
tar: Error is not recoverable: exiting now
The command "sudo tar xjf -.tar.bz2 --directory /" failed and exited with 2 during .
Your build has been stopped.
```

**Comment by fernandog** [comment]
@ratoaq2 have you decided yet about how to proceed with this?

ping @duramato

**Comment by ratoaq2** [comment]
Could you rebase it?

**Comment by fernandog** [comment]
@ratoaq2 done. merged latest develop

**Comment by fernandog** [comment]
@ratoaq2 no idea what's wrong and how to fix

```
guessit/test/test_yml.py:175: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
self = [Movies/Fear and Loathing in Las Vegas (1998)/Fear.and.Loathing.in.Las.Vegas.7...est Of Chuck Berry(2010)[320 Kbps]: OK!, Title Name [480p][1.5Mbps][.mp4]: OK!]
    def assert_ok(self):
        errors = [entry for entry in self if entry.error]
>       assert not errors
E       assert not [French Maid Services - Lola At Your Service WEB-DL SPLIT SCENES MP4-RARBG: ERROR! (valid=5, missing=0, different=1, extra=1, others=0)]
guessit/test/test_yml.py:108: AssertionError
```

It started to fail after the rebased. 
You have the rights to add commits to my branch.

**Comment by fernandog** [comment]
@ratoaq2 found the issue.  In this branch its detecting this streaming service for this filename:

```
guessit "French Maid Services - Lola At Your Service WEB-DL SPLIT SCENES MP4-RARBG"
For: French Maid Services - Lola At Your Service WEB-DL SPLIT SCENES MP4-RARBG
GuessIt found: {
    "title": "French Maid Services",
    "alternative_title": "Lola At Your Ser",
    "streaming_service": "Viceland",
    "source": "Web",
    "container": "mp4",
    "release_group": "RARBG",
    "type": "movie"
}
```

**Pull request merged**